onnxruntime/test/perftest/performance_runner.cc

```diff
@@ -76,16 +76,17 @@ bool PerformanceRunner::Initialize() {
     onnxruntime::kMklDnnExecutionProvider,
 #endif
 #ifdef USE_CUDA
+    #ifdef USE_TRT
```

Comment:
does this mean we've to enable both USE_CUDA and USE_TRT to use tensor rt? why?

Reply:
would be nice if --use_trt also flipped on cuda in the build.py script so you don't need to add both options.
(much like how --build_wheel internally sets --enable_pybind)